featurex/tests/test_extractors.py

```diff
@@ -86,6 +87,15 @@ def test_part_of_speech_extractor():
     assert df.iloc[1, 3] == 'NN'
     assert df.shape == (4, 4)
 
+def test_saliency_extractor(self):
+        image_dir = join(_get_test_data_path(), 'image')
+        stim = ImageStim(join(image_dir, 'apple.jpg'))
+        tl = stim.extract([SaliencyExtractor()])
+        ms = tl.data['SaliencyExtractor'].data['max_saliency']
+        assert ((ms >= 0) & (ms <= 1))
```

Comment:
These assertions should probably be more specific, and check against the (presumed) correct value. If you're worried about precision issues, you can use numpy's `almost_equal` to make sure values are within N digits of each other.